paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -241,7 +241,7 @@ class PaparazziPlugin : Plugin<Project> {
     }
   }
 
-  open class PaparazziTask : DefaultTask() {
+  internal open class PaparazziTask : DefaultTask() {
```

Comment:
Is there no case in which a consumer would need to do `tasks.withType(PaparazziTask)`? (And same q for `PrepareResourcesTask`.)

Reply:
no, these tasks are meant to be implementation detail.  consumers of plugins should use depend on task artifacts, not the task types themselves (although it's not always simple to get this right).

Reply:
I agree these are implementation details, but considering the amount of workarounds necessary to keep Paparazzi alive in a project, I would imagine there might be a need to adjust some properties on the task.

Reply:
fair point, reverting this

Reply:
> consumers of plugins should use depend on task artifacts, not the task types themselves

The best practice to get task artifacts are through properties on the task type since it gives you an implicit task dependency.

I would remove `open`, however, and consider something like `abstract` instead.

Reply:
changing to `abstract`